src/resolvers/user.ts

```diff
@@ -60,11 +63,13 @@ export function savedItems(
 export async function tags(
   parent: User,
   args: {
-    pagination: PaginationInput;
+    pagination: Pagination;
   },
   context: IContext
 ): Promise<TagConnection> {
-  validatePagination(args.pagination);
+  validatePagination(args.pagination,
```

Comment:
todo: should assign it back to args.pagination (or) `assign pagination = {first:default} when null` inside the [pagination module ](https://github.com/Pocket/apollo-utils/blob/main/src/pagination/validatePagination.ts#L10)